static/html/adminmenu.htm

```diff
@@ -1,15 +1,15 @@
 <div class="menu">
   <h3><u>Menu</u></h3>
-  <p><a href="/">Home</a></p>
-  <p><a href="/commands">Commands</a></p>
-  <p><a href="/stats">Stats</a></p>
-  <p><a href="/history/">Songlist History</a></p>
+  <p><a href="/">• Home</a></p>
```

Comment:
If the spacing is bad, you might want to use something like `reset.css` and this:
 
```css
/* place this at the start of your first css file. */
*, *:before, *:after {
  box-sizing: border-box;
} 
```

Check out `box-sizing` on Google, it's really great!